src/overlays/actors/ovl_En_Dekubaba/z_en_dekubaba.c

```diff
@@ -991,15 +991,15 @@ void EnDekubaba_ShrinkDie(EnDekubaba* this, PlayState* play) {
 
     if (Math_StepToF(&this->actor.scale.x, this->size * 0.1f * 0.01f, this->size * 0.1f * 0.01f)) {
         func_800286CC(play, &this->actor.home.pos, &sZeroVec, &sZeroVec, this->size * 500.0f, this->size * 100.0f);
-        if (this->actor.dropFlag == 0) {
+        if (this->actor.dropFlag == DROPFLAG_NONE) {
```

Comment:
`dropFlag` is a bitfield (btw, `dropFlags` (plural) ?), so I think 0 as "none" is enough